src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -866,6 +839,10 @@ InferenceProfiler::ValidLatencyMeasurement(
     }
   }
 
+  std::for_each(erase_indices.rbegin(), erase_indices.rend(), [this](size_t i) {
```

Comment:
Add a comment explaining why reverse index is being used.

Reply:
Added comment 👍